x2text-service/README.md

```diff
@@ -52,3 +52,27 @@ Response samples:
 status code : 200
 OK
 ```
+
+## Migration for x2text_audit Table to New Schema
+*(Applicable for users upgrading from versions before [v0.93.0](https://github.com/Zipstack/unstract/releases/tag/v0.93.0) to [v0.93.0](https://github.com/Zipstack/unstract/releases/tag/v0.93.0) or higher. This migration is not required for fresh installations or users already on v0.93.0 or a later version.)*
+
+### Migration Description
```

Comment:
@chandrasekharan-zipstack I can't see any places these data used, So it would not be a mandatory item. Also it only required those people using other than llm whisperer.

cc : @hari-kuriakose

Reply:
@muhammad-ali-e I would still suggest mentioning this doc in the migrating docs. As it is the migration guide is in a hard to reach place if the user misses the release notes that we show only once